pyop2/local_kernel.py

```diff
@@ -231,6 +220,16 @@ def dtypes(self):
     def dtypes(self, dtypes):
         self._dtypes = dtypes
 
+    @cached_property
+    def num_flops(self):
+        if self.flop_count is not None:
+            return self.flop_count
+        elif not configuration["compute_kernel_flops"]:
```

Comment:
I was wondering if we can pull this if clause into the `_compute` function where we call the logging of the flops to avoid having this in every specialised kernel, what do you think @connorjward ?